PFR/ForMathlib/Entropy/RuzsaDist.lean

```diff
@@ -876,7 +886,8 @@ lemma condRuzsaDist'_of_copy (X : Ω → G) {Y : Ω' → G} (hY : Measurable Y)
     simp only [A]
     rw [Measure.map_apply ‹_›]
     convert measure_empty
-    simp [← FiniteRange.range]
+    simp only [Finset.coe_union, ← FiniteRange.range, Set.compl_union, Set.preimage_inter,
+      Set.preimage_compl, Set.preimage_range, Set.compl_univ, Set.inter_empty]
     measurability
```

Comment:
When reviewing, I am not going to flag every single occurrence of the same issue. You should be proactive and fix them all when I show you one or two occurrences.

Reply:
Yes yes, sure. I was going to do exactly that. I'm sorry for the delay. I'll probably finish later today or tomorrow.

Reply:
Committed. Later today or tomorrow I'll give a global review to make sure your all your suggestions have been applied everywhere relevant.